wit/host.wit

```diff
@@ -2,8 +2,8 @@ package golem:api;
 
 world golem {
   import golem:api/host@0.2.0;
-  import golem:api/host@1.1.1;
-  import golem:api/oplog@1.1.1;
+  import golem:api/host@1.2.0;
```

Comment:
We have tests compiled against the 1.1.x wit. The fact that tests are still passing should be enough to prove that semver works here, but I can provide a dedicated example project if desired.